src/ec/packages/ec-component-footer-harmonised/docs/footer-harmonised.md

```diff
@@ -0,0 +1,91 @@
+# ECL Twig - EC Footer component
+
+npm package: `@ecl-twig/ec-component-footer-harmonised`
+
+```shell
+npm install --save @ecl-twig/ec-component-footer-harmonised
+```
+
+## Parameters
+
+- "group" (string) (default: 'group1'): 'group1' or 'group2'
+- "sections" (array of objects) (default: [])
+  - "title" (optional) (string) OR (object with Link component in property)
+  - "title_class_name" (optional) (string) (default: '')
+  - "description" (optional) (string) (default: '')
+  - "content_before" (optional) (string) (default: '')
+  - "list_class_name" (optional) (string) (default: '')
+  - "links" (optional) (array of Link components) (default: [])
+- "extra_classes" (optional) (string) (default: '') Extra classes (space separated) for the icon
+- "extra_attributes" (optional) (array) (default: []) Extra attributes for icon
+  - "name" (string) Attribute name, eg. 'data-test'
+  - "value" (string) Attribute value, eg: 'data-test-1'
+
+## Example:
+
+<!-- prettier-ignore -->
+```twig
+{% include '@ecl-twig/ec-component-site-footer-harmonised/footer-harmonised.html.twig' with { 
```

Comment:
where did this "ec-component-site-footer-harmonised" come out from..? ah, most likely the site header! :D

Reply:
Thx ! Updated in both component footer harmonised and standardised ;)